src/core/client/stream/tabs/comments/components/ConversationThread.tsx

```diff
@@ -63,7 +66,7 @@ const ConversationThread: StatelessComponent<
                   Show hidden comments
                 </Button>
               </Localized>
-              <Counter>{props.remaining}</Counter>
+              {props.remaining > 1 && <Counter>{props.remaining}</Counter>}
```

Comment:
Is this something that should be included in the `Localized` wrapper?

Reply:
Been thinking about that too, besides putting things ltr vs rtl was the only thing I could think of, but that should be addressed in a broader context.